module/Makefile

```diff
@@ -75,12 +75,21 @@ lint:
 ###                           Protobuf                                    ###
 ###############################################################################
 
+proto-all: proto-format proto-lint proto-gen
+
+proto-format:
+	@echo "Formatting Protobuf files"
+	$(DOCKER) run --rm -v $(CURDIR):/workspace \
+	--workdir /workspace tendermintdev/docker-build-proto \
+	find ./ -not -path "./third_party/*" -name *.proto -exec clang-format -i {} \;
+
+
 proto-gen:
 	@echo "Generating Protobuf files"
 	$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace tendermintdev/sdk-proto-gen:v0.1 sh ./contrib/local/protocgen.sh
 
 proto-lint:
-	@$(DOCKER_BUF) check lint --error-format=json
```

Comment:
what does this do?

Reply:
Not sure, brought it over from cosmos-sdk. https://docs.buf.build/lint-overview/